recipes/keystoneauth1/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{%set name = "keystoneauth1" %}
+{%set version = "2.8.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "7f92251600b86c368d2201317c60b82e37760e1399c223005710b1ee8a52b03f" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.8
+    - iso8601 >=0.1.11
+    - positional >=1.0.1
+    - requests >=2.10.0
+    - six >=1.9.0
+    - stevedore >=1.10.0
+
+test:
+  imports:
+    - keystoneauth1
+    - keystoneauth1.identity
+    - keystoneauth1.identity.v2
+    - keystoneauth1.identity.v3
+    - keystoneauth1.identity.generic
+    - keystoneauth1.extras
+
+about:
+  home: http://docs.openstack.org/developer/keystoneauth/
+  license: Apache Software License Version 2.0
```

Comment:
If you want to use `Apache 2.0` is OK, but the only real pre-requisite is that the license is easily identifiable.